deps/cares/cares.gyp

```diff
@@ -136,15 +136,14 @@
         }, {
           # Not Windows i.e. POSIX
           'cflags': [
-            '-g',
```

Comment:
This looks like a change that applies to other platforms as well. Was the -g flag left in by mistake by the Node.js team at some point? If so, I think we might want to split this into a separate PR since it's not strictly a z/OS change. Alternatively, we can leave it in for now and split it when we push to `master.zos`.

Reply:
Or we could do `'cflags!' : ['-g']` in the z/OS guard below so as not to affect other platforms

Reply:
That's what I thought (that it must've been left by mistake), so I left the 'remove -g' separated as in 'remove -g, and adjust cflags for z/OS'

I think let's leave it for now, and separate it into another PR for master.zos.

Reply:
Separated the 'remove -g' in the description for clarity.